templates/scenarios/csharp/message-extension/env/.env.dev.tpl

```diff
@@ -9,7 +9,4 @@ RESOURCE_SUFFIX=
 # Generated during provision, you can also add your own variables. 
```

Comment:
Please reference https://github.com/OfficeDev/TeamsFx-Samples/pull/731/files to add some comments to the beginning of all the .env files (note content of every .env file is slightly different).
By the way, the `.gitignore` file also needs update to only ignore `.env.xxx.user` and `.env.local`. Above sample PR also involves the change.

Reply:
Fixed

Reply:
Please also add comments to `.env.dev` and `.env.local` (note they're different)